inc/lightbox/self_test.php

```diff
@@ -137,7 +137,7 @@
             <?php if ( Util_Installed::redis() ): ?>
             <span style="background-color: #33cc33"><?php _e( 'Installed', 'w3-total-cache' ); ?></span>
             <?php else: ?>
-            <span style="background-color: #FFFF00"><?php _e( 'Not installed', 'w3-total-cache' ); ?></span>
+            <span style="background-color: #FF0000"><?php _e( 'Not installed', 'w3-total-cache' ); ?></span>
```

Comment:
Redis is not required and is not normally available on shared server plans.  This item should be marked yellow (`#FFFF00`) and "Not available".